### PR TITLE
BGDIINF_SB-1816: add more unit tests

### PR DESCRIPTION
**Pull request by hansmannj**

### CODE REVIEW

**Review by IsabelleBzr — APPROVED** [review]
Nice, thanks :+1: 
Just a minor change

File: app/__init__.py [hunk 1]
```diff
@@ -38,10 +38,21 @@ def add_cors_header(response):
         re.match(ALLOWED_DOMAINS_PATTERN, request.headers['Origin'])
     ):
         response.headers['Access-Control-Allow-Origin'] = request.headers['Origin']
-        response.headers['Access-Control-Allow-Methods'] = 'GET, POST, OPTIONS'
+        response.headers['Access-Control-Allow-Methods'] = 'GET, POST, PUT, OPTIONS'
```
[the remark this answers is not in the document]
Comment (hansmannj) [reply app/__init__.py:41]: Yes, that makes sense, thanks, I'll add that :-)

File: tests/samples/simple-kml.xml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-<root xmlns    = "https://www.exampel.ch/"
-      xmlns:py = "https://www.exampel.ch/">
-   <py:elem1 />
-   <elem2 xmlns="" />
-</root>
```
[the remark this answers is not in the document]
Comment (hansmannj) [reply tests/samples/simple-kml.xml:5]: :+1: Thanks!

**Review by IsabelleBzr — APPROVED** [review]
:+1:

**Review by ltkum — APPROVED** [review]
looks like solid work to me. Good job.

**Pull request merged**